### PR TITLE
FF128 Accept Header values string changed

### PR DESCRIPTION
**Pull request by hamishwillee**
FF128 This updates the accept header string to match changes in https://bugzilla.mozilla.org/show_bug.cgi?id=1711622

This is addition of  `image/svg+xml` and `image/png` to the `Accept` header in image/default requests.
The is also a preference to add `image/jxl` to the string which is disabled by default (another preference exists for enabling avif, but that is enabled by default and predates this, so not recorded).

I've added release note and experimental features updates too.

Related docs work can be tracked in #34047

### CODE REVIEW

File: files/en-us/web/http/content_negotiation/list_of_default_accept_values/index.md [hunk 1]
```diff
@@ -14,35 +14,41 @@ These are the values sent when the context doesn't give better information. Note
...
+| Firefox 64 and earlier [2] | `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`                                                                   |
 | Safari, Chrome             | `text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8`                                             |
-| Safari 5 [2]               | `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`                                                                   |
+| Safari 5 [3]               | `text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8`                                                                   |
 | Edge                       | `text/html, application/xhtml+xml, image/jxr, */*`                                                                                  |
 | Opera                      | `text/html, application/xml;q=0.9, application/xhtml+xml, image/png, image/webp, image/jpeg, image/gif, image/x-xbitmap, */*;q=0.1` |
 
-\[1] This value can be modified using the [`network.http.accept.default`](https://kb.mozillazine.org/Network.http.accept.default) parameter.
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply files/en-us/web/http/content_negotiation/list_of_default_accept_values/index.md:27]: Chasing up in https://bugzilla.mozilla.org/show_bug.cgi?id=1711622#c36

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 2]
```diff
@@ -73,6 +75,12 @@ This article provides information about the changes in Firefox 128 that affect d
 
 These features are newly shipped in Firefox 128 but are disabled by default. To experiment with them, search for the appropriate preference on the `about:config` page and set it to `true`. You can find more such features on the [Experimental features](/en-US/docs/Mozilla/Firefox/Experimental_features) page.
 
+Symmetrical spacing with CSS letter-spacing: layout.css.letter-spacing.model. The CSS letter-spacing property now splits the specified letter spacing evenly on both sides of each character. This is unlike the current behavior where spacing is added primarily to one side (Firefox bug 1891446).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/128/index.md:78]: Is this in the wrong PR/branch? Seems out of place :)

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 3]
```diff
@@ -2457,6 +2457,48 @@ The [`Clear-Site-Data`](/en-US/docs/Web/HTTP/Headers/Clear-Site-Data) HTTP respo
 
 ## HTTP
 
+### Accept header with MIME type image/jxl
+
+The HTTP [`Accept`](/en-US/docs/Web/HTTP/Headers/Accept) header in [default requests and image requests](/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values) can be configured via a preference to indicate support for the `image/jxl` MIME type. If enabled, this appears after `image/avif`.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:2462]: I wasn't sure about the last sentence "If enabled, this appears after `image/avif`".

Would it be clearer to say "If `image.avif.enabled` is set to `true`, then `image/avif` will take precedence over `image/jxl`."


Looks like this from the bug, I don't know if you want to include that:

| Prefs enabled                            | Image accept header                                                               |
| ---------------------------------------- | --------------------------------------------------------------------------------- |
| `image.jxl.enabled`, `image.jxl.enabled` | `image/avif,image/jxl,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5` |
| `image.avif.enabled`                     | `image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5`           |
| `image.jxl.enabled`                      | `image/jxl,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5`            |

**Review by bsmth — APPROVED** [review]
Some comments to help describe interactions with other prefs, but otherwise looking good to me. Thank you!

File: files/en-us/mozilla/firefox/releases/128/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/128/index.md:78]: Yes. It's a copy paste error. Will remove.

File: files/en-us/mozilla/firefox/releases/128/index.md [hunk 4]
```diff
@@ -73,6 +75,12 @@ This article provides information about the changes in Firefox 128 that affect d
...
+
+- **`image/jxl` MIME type in Accept header for default and image requests:** `image.jxl.enabled`.
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/128/index.md:80]:
```suggestion
These features are newly shipped in Firefox 128 but are disabled by default. To experiment with them, search for the appropriate preference on the `about:config` page and set it to `true`. You can find more such features on the [Experimental features](/en-US/docs/Mozilla/Firefox/Experimental_features) page.

- **`image/jxl` MIME type in Accept header for default and image requests:** `image.jxl.enabled`.
```

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 3 above)
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/experimental_features/index.md:2462]:
```suggestion
The HTTP [`Accept`](/en-US/docs/Web/HTTP/Headers/Accept) header in [default requests and image requests](/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values) can be configured via a preference to indicate support for the `image/jxl` MIME type.
```

Comment (hamishwillee) [reply files/en-us/mozilla/firefox/experimental_features/index.md]: Thanks @bsmth 

Just FYI ..

>> I wasn't sure about the last sentence "If enabled, this appears after image/avif".

> Would it be clearer to say "If image.avif.enabled is set to true, then image/avif will take precedence over image/jxl."

What I meant was that if `image.jxl.enabled` is enabled then `image/jxl` appears in the string, and it appears after `image/avif`.

To fix this I've removed the note as you don't really need to know this here.

> Looks like this from the bug, I don't know if you want to include that:

FYI I saw the bug report and my first instinct was to do as you mention - but when you think about it, that's contrary to the way we handle preferences and not easy to maintain.  The `image.avif.enabled` is enabled by default, so it is always true - this isn't something people turn on an off any more, even though they can. MDN doesn't track the preferences in and of themselves - we track them to highlight experimental features. So there is no point mentioning this at all.

The question them comes "where do we track `image.jxl.enabled` - the answer to that is ... "as an experimental feature". Which is what I have done :-).

**Pull request merged**